test/e2e/utils/handler.go

```diff
@@ -221,13 +221,17 @@ func generateTestSkip(testParams *TestParameters) string {
 	}
 
 	// TODO(songjiaxun) remove this logic after the next CSI driver release.
-	// TODO(saikatroyc) remove the skip bucket access checks when managed driver created with skip bucket access check support
 	if testParams.UseGKEManagedDriver {
-		skipTests = append(skipTests, "Pod.RestartPolicy.is.OnFailure$", "Job.with.RestartPolicy.OnFailure.eventually.succeed", "fast.termination", "fileCache", "gcsfuseIntegrationFileCache", "init.container", "istio", "csi-skip-bucket-access-check")
```

Comment:
We will need to wait for the roll out to complete, otherwise removing this will break managed deriver tests.